playbooks/engineer-workflow.md

```diff
@@ -142,6 +143,14 @@ Further reading:
 - [Rules of communicating at GitHub](https://ben.balter.com/2014/11/06/rules-of-communicating-at-github/)
 - [RFC on defining PR roles at Artsy](https://github.com/artsy/README/issues/177)
 
+### Giving Code Reviews
+
+The best PR comments are clear in their intention. While we don't mandate that PR comments follow any specific
+format, some people have found it helpful to adopt the standards outlined in
+[Conventional Comments](https://conventionalcomments.org).
+
+In particular, it helps with clarity and velocity to specify when a given comment is `blocking` or `non-blocking`.
```

Comment:
@damassi I agree w/ you re: blocking and non-blocking! Tried to call that out here but let me know if it could be stronger.

Reply:
This looks 👍 to me

Reply:
If the goal is to get people to do this then I think this language should be stronger, e.g. "In particular, we encourage the use of `blocking` and `non-blocking` to signal whether or not a question or comment should be resolved before merging."